swap/swap.go

```diff
@@ -318,6 +336,12 @@ func (s *Swap) handleEmitChequeMsg(ctx context.Context, p *Peer, msg *EmitCheque
 		return err
 	}
 
+	if err := p.Send(ctx, &ConfirmChequeMsg{
```

Comment:
@ralph-pichler the convention to use is:
```go
err := p.Send(...)
if err != nil {
 return err
}
```
for all error checks in the same block, this way you prevent allocating a new variable for every function call that checks for an error. please fix it in the calls below too

Reply:
I changed it for `handleConfirmChequeMsg` and `handleEmitChequeMsg`. Hope that's ok now, I remember the opposite recommendation from other PRs.